packages/screens/FeedNewArticle/FeedNewArticleScreen.tsx

```diff
@@ -167,7 +180,9 @@ export const FeedNewArticleScreen: ScreenFC<"FeedNewArticle"> = () => {
       // If the user uploaded files, but they are not pinned to IPFS, it returns files with empty url, so this is an error.
       if (formValues.files?.length && !remoteFiles.find((file) => file.url)) {
         console.error("upload file err : Fail to pin to IPFS");
-        setToastError({
+        setToast({
```

Comment:
suggestion: use try catch to centralize error processing

Reply:
I will not do that is this PR